test/integration/test-amp-pixel.js

```diff
@@ -35,6 +35,35 @@ describe.configure().skipIfPropertiesObfuscated().run('amp-pixel', function() {
     });
   });
 
+  describes.integration('amp-pixel integration test', {
+    body: `<amp-pixel src="${depositRequestUrl('has-referrer&title=TITLE')}">`,
+  }, env => {
+    it('should expand the TITLE macro', () => {
+      return withdrawRequest(env.win, 'has-referrer&title=AMP-TEST')
+          .then(request => {
+            expect(request.headers.referer).to.be.ok;
+          });
+    });
+  });
+
+  const requestUrl = depositRequestUrl('has-referrer&b=QUERY_PARAM(body)');
+  describes.integration('amp-pixel integration test', {
```

Comment:
every test needs a different name. otherwise mocha will do weird things.

and I suggest you move both macros in one test, so every test is testing one thing.

Reply:
Hmm they all already had the same name. Should I change the existing names as well?

Reply:
Done.